lib/tasks/orders.rb

```diff
@@ -0,0 +1,12 @@
+# This class used for additional methods of Orders
+class OrdersTask < OrdersController
+  def time_check
+    @orders = Order.where(status: 'waiting')
+    return unless @orders.nil?
+    @orders.each do |order|
+      if Time.now.utc - order.updated_at > 300
+        order.update(status: 'decline', driver_id: nil)
```

Comment:
`declined`, not `decline`.
And what about those orders which were not accepted in 10 minutes?